compute-budget/src/compute_budget_processor.rs

```diff
@@ -21,14 +22,15 @@ pub const MIN_HEAP_FRAME_BYTES: u32 = HEAP_LENGTH as u32;
 
 /// The total accounts data a transaction can load is limited to 64MiB to not break
 /// anyone in Mainnet-beta today. It can be set by set_loaded_accounts_data_size_limit instruction
-pub const MAX_LOADED_ACCOUNTS_DATA_SIZE_BYTES: u32 = 64 * 1024 * 1024;
+pub const MAX_LOADED_ACCOUNTS_DATA_SIZE_BYTES: NonZeroU32 =
+    unsafe { NonZeroU32::new_unchecked(64 * 1024 * 1024) };
 
 #[derive(Clone, Copy, Debug, PartialEq, Eq)]
 pub struct ComputeBudgetLimits {
     pub updated_heap_bytes: u32,
     pub compute_unit_limit: u32,
     pub compute_unit_price: u64,
-    pub loaded_accounts_bytes: u32,
+    pub loaded_accounts_bytes: NonZeroU32,
```

Comment:
I prefer to keep the `NonZeroU32` change in this PR since it ties in closely with the zero check which moved up earlier. The other values are allowed to be zero